### DISCUSSION TITLE
How to reference an organization repo?

### DISCUSSION THREAD

**Discussion by tballenger** [avitorio/outstatic#81] [Q&A]
Hey all, this looks like a great project, but unfortunately having issues integrating it into our organization. It is always prepending my personal github username when looking for my organization's repo --

eg I want it to point to `acme-corp/web` (which owns the OAUTH app) but it is always prepending my profile to the beginning of it despite pointing the OST_REPO_SLUG to `acme-corp/web` it is pointing to `tballenger/acme-corp/web` and throws an error when trying to save a collection, since the repo I'm pointing at is not at `tballenger/acme-corp/web` 

Any ideas? Thank you for the help!

**Comment by avitorio** [comment]
Hi @tballenger,

I believe you can use the `OST_REPO_OWNER` environment variable to make this work.

Here are all the possible environment variables for reference: https://outstatic.com/docs/environment-variables

Let me know if it works!
